### ISSUE TITLE
Desire for  a Variable Width Data Viz.

### ISSUE THREAD

**Issue by JazzfanRS** [shriprem/FWDataViz#52]
Would it be possible  to create a **variable** width data visualizer? I am new to GitHub, and to this plugin. I'm not a coder, but I think being able to specify characters present in every line as the column boundaries. (Example: A tab, a comma, etc) instead of fixed width would be quite useful as NPP does not seem to like to align text to visual columns.

Apologies if this isn't a correct place to post this 'Idea' and for not seeing if this had already been suggested.

**Comment by shriprem** [comment]
Have you tried the CSV Lint plugin?

I think in addition to the  comma separator, it also supports tab, semi-colon, pipe and some basic fixed width.

But beware: Notepad++ may crash if you open a CSV file with the CSV Lint plugin installed and then try to activate (i.e., display) the FWDataViz plugin side panel. There might be other incompatibilities too. Unfortunately, you cannot have both plugins active at the same time.

**Comment by JazzfanRS** [comment]
Okay thanks.

**Comment by JazzfanRS** [comment]
I barely know how to use regex. I don't code. I am learning how to use this plugin, and as you suggested CSV Lint plugin.

**Comment by shriprem** [comment]
There are programmers who know regex. And those who don't. In general, those who know regex are the sharper programmers.

When I have refactored legacy code, many times I have replaced over 25 lines with just a single line utilizing regex. So, learning regex is definitely worthwhile.

If you do a web search for regex helper sites, you will find plenty of excellent choices. If you are looking for Windows-based tools, [Jan Goyvaerts](https://www.jgsoft.com) is your man. Besides PowerGREP, RegexBuddy & RegexMagic, all his other offerings have a solid support for regex. He has authored a few excellent books on regex as well.
